extensions/wikia/UserRenameTool/SpecialRenameuser_body.php

```diff
@@ -84,7 +84,7 @@ public static function validateData( array $data, User $user, bool $selfRename =
 			$errorList[] = 'userrenametool-error-non-alphanumeric';
 		}
 
-		if ( strlen( $newUsername ) > \RenameUserHelper::MAX_USERNAME_LENGTH ) {
+		if ( strlen( $newUsername ) > \RenameUserProcess::MAX_USERNAME_LENGTH ) {
```

Comment:
Use `strlen` to measure byte length and `mb_strlen` to measure character length of UTF-8 strings.  Judging by the context, the latter one fits here better.